web/src/main/webapp/WEB-INF/config-summary.xml

```diff
@@ -137,19 +137,19 @@
 
   <summaryTypes>
     <summaryType name="details" format="DIMENSION">
-      <item facet="type" translator="codelist:gmd:MD_ScopeCode"/>
+      <item facet="type" sortBy="label" sortOrder="asc" translator="codelist:gmd:MD_ScopeCode"/>
       <item facet="mdActions"/>
-      <item facet="topicCat" translator="codelist:gmd:MD_TopicCategoryCode" max="20"/>
+      <item facet="topicCat" sortBy="label" sortOrder="asc" translator="codelist:gmd:MD_TopicCategoryCode" max="20"/>
       <item facet="inspireThemeURI" sortBy="value" sortOrder="asc" max="35"
             translator="term:http://inspire.ec.europa.eu/theme"/>
       <!--<item facet="inspireTheme" sortBy="value" sortOrder="asc" max="35"/>-->
       <!--<item facet="inspireThemeWithAc" sortBy="value" sortOrder="asc" max="35"/>-->
-      <item facet="keyword" max="15"/>
-      <item facet="orgName" max="25"/>
+      <item facet="keyword" sortBy="label" sortOrder="asc" max="15"/>
+      <item facet="orgName" sortBy="label" sortOrder="asc" max="25"/>
       <item facet="sourceCatalog"
             translator="db:org.fao.geonet.repository.SourceRepository:findOneByUuid"/>
       <item facet="createDateYear" sortBy="value" sortOrder="desc" max="40"/>
-      <item facet="format" max="15" sortBy="value" sortOrder="asc"/>
+      <item facet="format" max="15" sortBy="label" sortOrder="asc"/>
       <item facet="spatialRepresentationType"
```

Comment:
Some of the text filters seem still sorted by count.

Reply:
@josegar74 

Would you hold for the merge? I am gonna keep identifying the alphabetic items as much as possible. Then update with this pull request. Thanks